### PR TITLE
chore: update outdated links

### PR DESCRIPTION
**Pull request by tatomyr**
## What/Why/How?

- [x] Updated the outdated links that pointed to the `redoc.ly` domain.
- [x] Updated dependencies.

## Check yourself

- [ ] Code changed? - Tested with redoc/reference-docs/workflows (internal)
- [ ] All new/updated code is covered with tests
- [ ] New package installed? - Tested in different environments (browser/node)

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: .github/styles/Redocly-Docs/FutureTense.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 extends: existence
 message: 'Avoid using future tense: "%s". Use present tense instead.'
-link: https://intranet.redoc.ly/contributing/documentation-style-guide/#tone-and-audience
+link: https://intranet.redocly.com/contributing/documentation-style-guide/#tone-and-audience
```
Comment (tatomyr) [inline .github/styles/Redocly-Docs/FutureTense.yml:3]: @lornajane could you check if the links are correct? None of them work for me, and the same goes for the neighbouring files.

File: packages/cli/src/commands/build-docs/utils.ts [hunk 2]
```diff
@@ -87,7 +87,7 @@ export async function getPageHTML(
 
       </script>`,
     redocHead:
-      `<script src="https://cdn.redoc.ly/redoc/v${redocCurrentVersion}/bundles/redoc.standalone.js"></script>` +
+      `<script src="https://cdn.redocly.com/redoc/v${redocCurrentVersion}/bundles/redoc.standalone.js"></script>` +
```
Comment (tatomyr) [inline packages/cli/src/commands/build-docs/utils.ts:90]: @AlexVarchuk could you check if this is a correct replacement?

File: .github/styles/Redocly-Docs/FutureTense.yml (hunk 1 above)
Comment (lornajane) [reply .github/styles/Redocly-Docs/FutureTense.yml:3]: I guess we moved something? The correct link is https://intranet.redocly.com/content/documentation-style-guide#tone-and-audience

**Review by AlexVarchuk — APPROVED** [review]
The links look correct and they are available now

**Review by lornajane — APPROVED** [review]

**Pull request merged**